### PR TITLE
feat: Datepicker component

### PR DESCRIPTION
**Pull request by denkristoffer**
# Purpose of PR

Umbrella PR to contain all work on the coming Datepicker component

### CODE REVIEW

File: packages/components/datepicker/README.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: 'Datepicker'
+status: 'alpha'
```
Comment (denkristoffer) [inline packages/components/datepicker/README.mdx:3]: Is the API stable? Then it's beta, no? 😁

File: packages/components/datepicker/README.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+isNew: true
+slug: /components/datepicker/
+github: 'https://github.com/contentful/forma-36/tree/main/packages/components/datepicker'
+storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-datepicker'
```
Comment (denkristoffer) [inline packages/components/datepicker/README.mdx:7]: Should we link the Chromatic storybook?

File: packages/components/datepicker/examples/Calendar/Indicator.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  tomorrow.setDate(tomorrow.getDate() + 1);
+  const yesterday = new Date();
+  yesterday.setDate(yesterday.getDate() - 1);
+
+  const daysWithSchedule = [yesterday, tomorrow];
+  const [selectedDay, setSelectedDay] = useState(today);
+
+  const styles = {
+    daysWithSchedule: css({
+      '&:before': {
```
Comment (denkristoffer) [inline packages/components/datepicker/examples/Calendar/Indicator.tsx:18]:
```suggestion
      '&::before': {
```

**Review by denkristoffer — COMMENTED** [review]
LGTM 🚢

File: packages/components/datepicker/README.mdx (hunk 1 above)
Comment (bgutsol) [reply packages/components/datepicker/README.mdx:3]: don't even start 😄

**Review by bgutsol — APPROVED** [review]

**Pull request merged**